src/lib/redux-store/explore/bootstrapMetricsExplorer.ts

```diff
@@ -0,0 +1,25 @@
+import { createAsyncThunk } from "$lib/redux-store/redux-toolkit-wrapper";
+import { EntityType } from "$common/data-modeler-state-service/entity-state-service/EntityStateService";
+import { bootstrapMetricsDefinition } from "$lib/redux-store/metrics-definition/bootstrapMetricsDefinition";
+import type { RillReduxState } from "$lib/redux-store/store-root";
+import { selectMetricsExplorerById } from "$lib/redux-store/explore/explore-selectors";
+import { syncExplore } from "$lib/redux-store/explore/explore-apis";
+import { selectDimensionsByMetricsId } from "$lib/redux-store/dimension-definition/dimension-definition-selectors";
+import { selectMeasuresByMetricsId } from "$lib/redux-store/measure-definition/measure-definition-selectors";
+
+export const bootstrapMetricsExplorer = createAsyncThunk(
```

Comment:
what do we mean by bootstrap here? I am finding this language confusing when I see it. Does this mean that we're syncing everything needed to make the explore up-to-date?

Reply:
It loads dimensions and measures if not already loaded. It also loads leaderboards, time series and big numbers. Hence the name, bootstrap everything related to the explorer.